docs/containerd-env-setup.md

```diff
@@ -30,7 +30,7 @@ $ sudo tee /etc/nydusd-config.json > /dev/null << EOF
       "type": "registry",
       "config": {
         "scheme": "",
-        "skip_verify": false,
+        "skip_verify": true,
```

Comment:
Um. I am not sure disabling TLS verify by default is an appropriate choice.

Reply:
We have given an explanation about this below.